### PR TITLE
Update to Blazor 3.0.100-preview8-013656

### LINKED ISSUE #9: Update to preview 6
Please update to preview 6

### PR DESCRIPTION
**Pull request by campersau**
fixes #9

### CODE REVIEW

File: src/Blazor.Extensions.Notifications.JS/src/NotificationService.ts [hunk 1]
```diff
@@ -19,8 +19,7 @@ export class NotificationsManager implements INotificationsManager {
     return false;
   }
 
-  public Create = (title: string, options: object): object => {
-    var note = new Notification(title, options);
-    return note;
+  public Create = (title: string, options: object): void => {
+    new Notification(title, options);
```
Comment (campersau) [inline src/Blazor.Extensions.Notifications.JS/src/NotificationService.ts:23]: `System.Text.Json` throws an exception serializing this object which is visible in the browser console. So we don't return it.

**Comment by vertonghenb** [comment]
@campersau looks great, thanks for the PR!
Is it also possible to update the readme to include the .js file for client-side and server side blazor?

**Comment by campersau** [comment]
@vertonghenb Sorry, I am not sure I undestand. The .js file will be included automatically. Why or what exactly should be in the readme about it?

**Comment by vertonghenb** [comment]
adding _content/BlazorNotifications  to the _host file when on server-side blazor
check https://github.com/Blazored/Typeahead for an example

**Comment by campersau** [comment]
Blazor moved to a new Razor Class Library structure, which requires static files inside the `wwwroot` folder and sadly they can't be linked. So I have merged the JS project into the library. Or do you know other workarounds?

See https://docs.microsoft.com/en-us/aspnet/core/blazor/class-libraries?view=aspnetcore-3.0&tabs=visual-studio and https://docs.microsoft.com/en-us/aspnet/core/razor-pages/ui-class?view=aspnetcore-3.0&tabs=visual-studio#create-an-rcl-with-static-assets

**Pull request closed without merging**